Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM gcr.io/distroless/static-debian11:nonroot
-ENTRYPOINT ["/baton-beeline"]
+FROM public.ecr.aws/lambda/provided:al2023
```

Comment:
I think we are supposed to have a `Dockerfile` and `Dockerfile.lambda`.

Reply:
Yes I forgot this in my own change, thank you Lauren.